src/styles/nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+    cursor: pointer;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    font-weight: 500;
+    text-transform: uppercase;
+    font-size: 12px;
+    height: 60px;
+    display: flex;
+    align-items: center;
+    font-family: "Roboto", Arial, sans-serif;
+    color: inherit;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link:hover {
+    color: $color-accent-blue;
+    transition: color 300ms ease-in;
+  }
+
+  .is-active {
+    position: relative;
+    color: $color-accent-blue;
+  }
+
+  &__is-active:after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+    background-color: $color-accent-blue;
+  }
+
+}
```

Comment:
It`s doesn`t work, use SASS nesting
![image](https://github.com/mate-academy/layout_catalog/assets/64101898/41522029-409a-4de7-9931-2b481608401e)

```suggestion
  .is-active {
    position: relative;
    color: $color-accent-blue;
    
    &:after {
      content: "";
      position: absolute;
      bottom: 0;
      left: 0;
      width: 100%;
      height: 4px;
      border-radius: 8px;
      background-color: $color-accent-blue;
    }
  }
}
```